src/Compatibility/Microsoft.DotNet.PackageValidation/NupkgParser.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    /// <summary>
+    /// Creates a package object from the nupkg package.
+    /// </summary>
+    public class NupkgParser
+    {
+        public static Package CreatePackage(string packagePath, string runtimeGraph)
```

Comment:
Do we need to add docs for all public API?

Reply:
yes, i will add those.